client-mvp-04/src/routes/index.js

```diff
@@ -21,6 +22,11 @@ export const Routes = [
     key: 'dev-ui-kit',
     component: DevUiKit,
   },
+  {
+    path: '/register',
+    key: 'create-account-link',
+    component: Register,
```

Comment:
In MVP (v0.4) Requirements page name is `Create Account`. 

Let's change on:
    path: '/create-account',
    key: 'create-account',
    component: CreateAccount,